src/backend/joanie/core/api/admin.py

```diff
@@ -437,6 +437,20 @@ class OrderViewSet(
     )
     ordering = "created_on"
 
+    @action(detail=True, methods=["POST"])
+    def cancel(self, request, *args, **kwargs):
+        """Change the state to cancelled if the order is not yet validated."""
+        order = self.get_object()
+
+        if order.state == enums.ORDER_STATE_VALIDATED:
+            return Response(
+                "Cannot cancel a validated order.",
+                status=HTTPStatus.UNPROCESSABLE_ENTITY,
+            )
```

Comment:
I will rectify this right away. I will take this condition away and adjust my tests.
I thought that it would involve other actions 'refund/unenrollment' by changing the state of a 'validated' order.